entrypoint.sh

```diff
@@ -15,7 +15,10 @@ if [[ ! "${METRICS_RESOLUTION:-1s}" =~ ^[1-5]s$ ]]; then
 fi
 sed -i "s/1s/${METRICS_RESOLUTION:-1s}/" /etc/prometheus.yml
 sed -i "s/ENV_METRICS_RETENTION/${METRICS_RETENTION:-720h}/" /etc/supervisord.d/pmm.ini
-sed -i "s/ENV_METRICS_MEMORY/${METRICS_MEMORY:-262144}/" /etc/supervisord.d/pmm.ini
+
+# Prometheus memory PMM-969
+METRICS_MEMORY_MULTIPLIED=$(( ${METRICS_MEMORY:-262144} * 1024 ))
```

Comment:
Sorry, can you please explain to me how this works? AFAIU, bash can't just multiply numbers like this.